lib/gcloud/datastore/dataset/query_results.rb

```diff
@@ -60,24 +60,21 @@ class QueryResults < DelegateClass(::Array)
         # Convenience method for determining id the more_results value
         # is "NOT_FINISHED"
         def not_finished?
-          more_results == Proto.to_more_results_string(
-            Proto::QueryResultBatch::MoreResultsType::NOT_FINISHED)
+          more_results == :NOT_FINISHED
         end
 
         ##
         # Convenience method for determining id the more_results value
         # is "MORE_RESULTS_AFTER_LIMIT"
         def more_after_limit?
-          more_results == Proto.to_more_results_string(
-            Proto::QueryResultBatch::MoreResultsType::MORE_RESULTS_AFTER_LIMIT)
+          more_results == :MORE_RESULTS_AFTER_LIMIT
         end
 
```

Comment:
There is one more enum value now:
more_results_after_cursor. This gets set if you do a query with an end_cursor